github_releaser/github_releaser.py

```diff
@@ -140,9 +140,6 @@ def push_files(self, branch_name: str, dest_dir: str, message: str, files: List[
             raise ArgumentError(
                 "cannot push more than {} files".format(MAX_UPLOAD))
 
-        if not branch_name:
```

Comment:
@AlieksandraK It is because the option didn't set the default value:
```python
@click.option("--branch-name", "--b", default="master", help="The branch name.")
```

Reply:
ah, wonderful then!